api.planx.uk/server.ts

```diff
@@ -421,11 +247,20 @@ app.use(
   }),
 );
 
+passport.use("google", googleStrategy);
+
+passport.serializeUser(function (user, cb) {
+  cb(null, user);
+});
+
+passport.deserializeUser(function (obj: Express.User, cb) {
+  cb(null, obj);
+});
 app.use(passport.initialize());
 app.use(passport.session());
 app.use(urlencoded({ extended: true }));
 
-app.use("/auth", router);
+app.use(authRoutes);
```

Comment:
really satisying diff in this file :muscle:

Reply:
💯